client/src/App.js

```diff
@@ -191,7 +193,7 @@ class App extends Component {
                           navItems={alumniNavBarItems()}
                           activeItem={'home'}
                       />
-                      <div> Home! </div>
+                      <div> Home! Welcome {this.props.details && this.props.details.name} ({this.props.role})</div>
```

Comment:
We are able to use `details` and `role` that was added to `store` in `LoginForm` in this manner!